src/index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>jsBowling</title>
+  <link rel="stylesheet", href="style.css">
+</head>
+<body>
+  <script
+  src="https://code.jquery.com/jquery-3.5.1.min.js"
+  integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+  crossorigin="anonymous"></script>
+
+  <div class="container">
+    <div class="score-display">
+    <table>
+      <thead>
+      <tr>
+        <th>Frame</th>
+        <th>Roll One</th>
+        <th>Roll Two</th>
+        <th></th>
+        <th>Current Score</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td>1.</td>
+        <td><input class="f1r1" type="number" min="0" max="10"></td></td>
+        <td><input class="f1r2" type="number" min="0" max="10"></td></td>
+        <th><button id="roll-input-one">Add Scores</button></th>
```

Comment:
I wonder if relying on the user to hit the 'Add Scores' button is necessary? What will happen if they forget to hit this button